meta-lmp-base/recipes-containers/skopeo/skopeo_git.bb

```diff
@@ -24,6 +24,7 @@ SRC_URI = " \
     file://storage.conf \
     file://registries.conf \
     file://0001-makefile-add-GOBUILDFLAGS-to-go-build-call.patch \
+    file://0001-cmd-copy-max-parallel-pulls-param.patch \
```

Comment:
place this patch on a `skopeo_git.bbappend` will simplify next updates and we can keep the recipe aligned with the meta-virt layer.

```
FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"

SRC_URI += "file://0001-cmd-copy-max-parallel-pulls-param.patch"
```

Reply:
Something like this?

```
--- a/meta-lmp-base/recipes-containers/skopeo/skopeo_git.bbappend
+++ b/meta-lmp-base/recipes-containers/skopeo/skopeo_git.bbappend
@@ -1,3 +1,6 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-SRC_URI += "file://docker-config-support-default-system-config.patch;patchdir=src/import"
+SRC_URI += " \
+    file://docker-config-support-default-system-config.patch;patchdir=src/import \
+    file://0001-cmd-copy-max-parallel-pulls-param.patch;patchdir=src/import \
+"

```